### ISSUE TITLE
Specifying Resource Usage for Merge Jobs

### ISSUE THREAD

**Issue by klannon** [matz-e/lobster#278]
It's probably fine to hard-code memory, but we should either allow users to set the disk usage needs for merge jobs, or we should estimate them from things that Lobster knows.  Ingredients include:

1.  Sandbox size
2.  Input size: Could be equal to requested output size (see below) but do we need to worry about ROOT file overhead in size?  E.g. for a 1 GB output, the inputs might be 1 GB + epsilon if we're merging a small number of files, or it might be 2.5 GB if we're merging hundreds of tiny files.  (At least that's how I understand it.)
3.  Output size:  User specifies this number in the config.

**Comment by matz-e** [comment]
If we're really worried about ROOT overhead, we could probably include some estimate for overhead, or pull it from the database.

**Comment by klannon** [comment]
I thought @khurtado added a step to the merging where we look at the size of each file we want to merge and estimate how much is real data vs overhead in constructing merge jobs.  Am I remembering correctly?

**Comment by matz-e** [comment]
Yes, that's I was referring to. Or we could look at the data collected so far and get an educated estimate.

**Comment by matz-e** [comment]
FWIW, this may be something for @WenzhaoLi?
